src/plugins/physics/EC_RigidBody_Ammo.js

```diff
@@ -0,0 +1,753 @@
+            case EC_RigidBody.ShapeType.HeightField:
+            case EC_RigidBody.ShapeType.ConvexHull:
+                return false;
+                break;
+            default:
+                return true;
+        }
+    },
+    
+    /**
+        Create the body. No-op if the scene is not associated with a physics world.
+        
+        @method createBody
+    */
+    createBody : function()
+    {
+        if (this.parentEntity.placeable === undefined ||
+            this.parentEntity.placeable === null)
+            return;
+
+        this.dirty_ = false;
+        
+        // Realase the old body
+        this.removeCollisionShape();
+        this.removeBody();
+
+        this.createCollisionShape();
+        if (this.collisionShape_ === null)
+            return;
+        
+        // Read component's attribute values
+        var mass           = this.attributes.mass.get();
+        var linDamping     = this.attributes.linearDamping.get();
+        var angDamping     = this.attributes.angularDamping.get();
+        var collisionLayer = this.attributes.collisionLayer.get();
+        var collisionMask  = this.attributes.collisionMask.get();
+        var friction       = this.attributes.friction.get();
+        var linVel         = this.attributes.linearVelocity.get();
+        var angVel         = this.attributes.angularVelocity.get();
+        var rolFri         = this.attributes.rollingFriction.get();
+        var linFactor      = this.attributes.linearFactor.get();
+        var angFactor      = this.attributes.angularFactor.get();
+
+        var isKinematic    = this.attributes.kinematic.get();
+        var isPhantom      = this.attributes.phantom.get();
+        var drawDebug      = this.attributes.drawDebug.get();
+
+        var isDynamic = this.isDynamic();
+        
+        // Read placeables position and rotation
+        var pos = this.parentEntity.placeable.worldPosition();
+        var rot = this.parentEntity.placeable.worldOrientation();
+        var transform = new Ammo.btTransform();
+        var position = new Ammo.btVector3(pos.x, pos.y, pos.z);
+        var quat = new Ammo.btQuaternion(rot.x, rot.y, rot.z, rot.w);
+        transform.setOrigin(position);
+        transform.setRotation(quat);
+        
+
+        var localInertia = new Ammo.btVector3(0.0, 0.0, 0.0);
+        if (isDynamic)
+            this.collisionShape_.calculateLocalInertia(mass, localInertia);
+        
+        var myMotionState = new Ammo.btDefaultMotionState(transform);
+        var rbInfo = new Ammo.btRigidBodyConstructionInfo(mass,
+                                                          myMotionState,
+                                                          this.collisionShape_,
+                                                          localInertia);
+        rbInfo.set_m_linearDamping(linDamping);
+        rbInfo.set_m_angularDamping(angDamping);
+        
+        this.rigidbody_ = new Ammo.btRigidBody(rbInfo);
```

Comment:
I would maybe name this `ammoRigidBody` or `rigidAmmo` etc. Got a bit confused by the code until searched this line :)